src/plugin/road_network.cc

```diff
@@ -46,6 +47,9 @@ class RoadNetworkLoader : public maliput::plugin::RoadNetworkLoader {
       const std::map<std::string, std::string>& properties) const override {
     return malidrive::builder::RoadNetworkBuilder(properties)();
   }
+  std::map<std::string, std::string> GetDefaultParameters() const override {
+    return malidrive::builder::params::GetDefaultParameters();
```

Comment:
Can't we do the same as in this function but here? It seems a trivial thing and we are adding a cc file to the library just for one line (without the boilerplate code).

Reply:
Done.